app/src/main/java/io/blockv/example/BlockvModule.java

```diff
@@ -7,7 +7,7 @@
 import io.blockv.core.client.manager.*;
 import io.blockv.example.utils.ImageFaceView;
 import io.blockv.face.client.FaceManager;
-import io.blockv.faces.NativeImageFace;
+import io.blockv.face3d.Face3D;
```

Comment:
Can you rename your Face3D package to io.blockv.faces to keep it inline with the other native faces